homeassistant/components/sensor/coinmarketcap.py

```diff
@@ -42,7 +44,8 @@
 SCAN_INTERVAL = timedelta(minutes=15)
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
-    vol.Optional(CONF_CURRENCY, default=DEFAULT_CURRENCY): cv.string,
+    vol.Optional(CONF_CURRENCY_ID, default=DEFAULT_CURRENCY_ID):
+        cv.positive_int,
```

Comment:
I also need feedback on this. I looked through the config validation types and could only find a type "positive" integer. However this seems rather like a non-negative integer (as min is set to `min=0`), the API requires real positive integers (`>0`). 

Should I rather make a new local type for this specific case, or is it okay if I go through the project and replace existing types with `non_negative_int`?:
```
non_negative_int = vol.All(vol.Coerce(int), vol.Range(min=0))
positive_int = vol.All(vol.Coerce(int), vol.Range(min=1))
```

Reply:
Cheers to the quick feedback!